app/assets/javascripts/application.js

```diff
@@ -28,3 +28,16 @@
 //= require pickers
 //= require cookies_eu
 //= require cocoon
```

Comment:
I think this should be put in it's own javascript file. (I like how this critique applies so much to me as well... ;) )